app/views/prison/visits/_issue_with_prisoner_section.html.erb

```diff
@@ -4,10 +4,14 @@
       <%= error_container(f, :selection) do %>
         <label class="block-label">
           <%=
-            f.radio_button(
-              :selection, :no_allowance,
-              class: 'js-Conditional',
-              data: { 'conditional-el': 'no_allowance_details' }
+            f.check_box(
```

Comment:
I think we need to preserve the checked boxes when the page gets re-rendered after a validation error, at the moment if a validation error happens any checkboxes that the user has selected will be lost. I haven't tried but I think that will be the case...